nodecg/Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM nodecg/nodecg:latest
 # Copy Bundle
 COPY bundles /usr/src/app/bundles
 
+# A hack to get the bower install to work.  Something to do with .git files confusing it all. Meh.. https://github.com/bower/bower/issues/1492
+RUN cd /usr/src/app/bundles/thebiggame && rm .git .gitignore .gitattributes
```

Comment:
I personally think this should be handled with a `.dockerignore` file, for two reasons: first, it means these files don't end up in the image at all, saving layer size; and second, it doesn't require an extra RUN layer.

Documentation on `.dockerignore` here: https://docs.docker.com/engine/reference/builder/#dockerignore-file

Reply:
Yeah good idea, i'll get on it